### PR TITLE
[PLAYER-3954] Enable in manifest closed captions

### PR DESCRIPTION
**Pull request by orobles-ooyala**

### CODE REVIEW

File: sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinPlayerObserver.java [hunk 1]
```diff
@@ -218,11 +220,27 @@ private void bridgeAdPodStartedNotification() {
   }
 
   private void bridgeLiveCCChangedNotification(Object data) {
+    Map<String, String> map = (Map<String, String>) data;
```
Comment (brol1dev) [inline sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinPlayerObserver.java]: can `data` be null here too? Better check, I think.

**Review by brol1dev — COMMENTED** [review]
Added a question. I know is not code that you touched, but the one you created has a null check which I think should be added to the other one too.

File: sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinPlayerObserver.java (hunk 1 above)
Comment (orobles-ooyala) [reply sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinPlayerObserver.java]: Done

**Review by brol1dev — APPROVED** [review]
LGTM

**Review by akshaydch11 — APPROVED** [review]

**Pull request merged**